src/utilities/parsing.ts

```diff
@@ -61,7 +61,11 @@ export function findDeep(obj: any, pred: (obj: any) => boolean) {
 
 // parsing filter string for TableComponent filter function
 function getNumberFromFilterString(filterString: string): number {
-    return Number(filterString.replace(/[^0-9.+-.]+/g, ""));
+    const n = filterString.replace(/[^0-9.+-.]+/g, "");
```

Comment:
can we not turn this into a single regex with the comparison operator? @confluence is our regex expert here :laughing:

Reply:
This regex will return the numerical part from the filter string. Not sure what we should do at here 🤔